### PR TITLE
Initial repository setup

### PR DESCRIPTION
**Pull request by james-wallis**
The initial code for this:
1. Sets up the Next.js app with Tailwind CSS and Radix UI, hosted on Vercel
2. Storybook with Chromatic deploy for component testing
3. Jest and React Testing Library for unit tests
4. ESLint, Prettier, Lint Staged and Husky for code standard
5. VSCode settings, some recommended extensions

* [x] Feature

#### Jira

https://ostmodern.atlassian.net/browse/SL-2251

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    "prettier-plugin-tailwindcss": "^0.1.13",
+    "storybook-addon-next-router": "^4.0.1",
+    "ts-jest": "^29.0.3",
+    "typescript": "4.8.4",
+    "util": "^0.12.5"
+  },
+  "resolutions": {
+    "webpack": "^5"
+  },
+  "readme": "ERROR: No README data found!",
```
Comment (freixedelo) [inline package.json:78]: is this expected ?

File: src/components/navigation/bar/bar.component.tsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    <div className="flex h-14 w-full items-center justify-between bg-nav-bar px-8 font-sans text-sm text-black lg:h-16 lg:px-14">
+      <div className="flex h-full items-center gap-8 font-bold md:gap-12">
+        <div className="flex">
+          <Image src={Logo} alt="Skylark Logo" width="30" height="30" />
+          <p className="ml-3 hidden font-heading text-xl md:block lg:ml-4">
+            Skylark
+          </p>
+        </div>
+        {apps.map(({ text, href }) => (
+          // Use legacyBehaviour so we can style the a tag
```
Comment (freixedelo) [inline src/components/navigation/bar/bar.component.tsx:41]: 😮

**Review by freixedelo — APPROVED** [review]

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,39 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```
[the remark this answers is not in the document]
Comment (james-wallis) [reply .gitignore:1]: Just used the Next.js one

File: package.json (hunk 1 above)
Comment (james-wallis) [reply package.json:78]: I have no idea where the readme part has come from, removing

File: src/components/navigation/bar/bar.component.tsx (hunk 2 above)
Comment (james-wallis) [reply src/components/navigation/bar/bar.component.tsx:41]: ikr

**Pull request merged**